package LoginProto;

message Login{
	required string name = 1;
	required string pwd = 2;
}
